app/assets/stylesheets/reference.scss

```diff
@@ -136,14 +136,21 @@ h3.plumbing {
     @include border-bottom-left-radius(3px);
     @include border-bottom-right-radius(3px);
 
-    input {
-      width: 70px;
-      padding: 1px 3px;
-      font-size: 11px;
-      color: $font-color;
-      background-color: #f0efe8;
-      border: solid 1px #e2e0d8;
-      @include border-radius(1px);
+    p {
+      // `all` and `display` are for overriding the global style defined
+      // for <p> at the beginning of "typography.scss".
+      all: unset;
+      display: block;
+      color: $light-font-color;
+    }
+
+    code {
+      user-select: all;
+    }
+
+    span {
```

Comment:
A bare `span` seems a bit broad here (even though it is scoped to the dropdown). Would we want `span.prompt` or something?

Reply:
That makes sense actually. I figured there is only one place with ID `previous-versions-dropdown`, hence no need to go more specific, but adding a class would improve readability as well.

Reply:
Done!